react/features/security/components/security-dialog/web/SecurityDialog.js

```diff
@@ -125,14 +125,16 @@ function mapStateToProps(state) {
     } = state['features/base/conference'];
     const { roomPasswordNumberOfDigits } = state['features/base/config'];
 
+    const showE2ee = Boolean(e2eeSupported) && isLocalParticipantModerator(state);
+
     return {
         _canEditPassword: isLocalParticipantModerator(state),
         _conference: conference,
         _dialIn: state['features/invite'],
         _locked: locked,
         _password: password,
         _passwordNumberOfDigits: roomPasswordNumberOfDigits,
-        _showE2ee: Boolean(e2eeSupported)
+        _showE2ee: showE2ee
```

Comment:
Why not just `_showE2ee: Boolean(e2eeSupported) && isLocalParticipantModerator(state)` instead of an extra const?